receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -146,6 +146,12 @@ attributes:
     enum:
       - primary_shards
       - total
+  get_result:
+    value: result
+    description: Result of get operation
+    enum:
+      - exists
+      - missing
```

Comment:
`exists` and `missing` are the names used in JSON. I tried looking into the documentation and didn't found these terms in use for `get` (especially, `exists` was used in regards to `head`, so it might be a good idea to change it to `hit`), so I'm fine with changing these name.